client/src/components/breadcrumbs.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import './Breadcrumbs.css'
+
+// internal logic for breadcrumbs:
+// "/"   -->  ["home"]
+// "/one" --> ["home", "one"]
+// "/one/one_one" --> ["home", "one", "one_one"] --> home / one / one_one
+//
+// hint:
+// ["home", "one", "one_one"].join("/")
+
+const Breadcrumbs = ({ currentPath, fetchFiles }) => {

```

Comment:
could you please rename file for this component and make start with the capital "B": Breadcrumbs.jsx

Reply:
I did so, but when I start the studio, it's lowercase again....

Reply:
try closing all files in VS Code,
but keep VS Code open
then try renaming file inside VS

Reply:
name for this file still starts with small "s" ;)